src/dev-app/platform/platform-demo.html

```diff
@@ -1,12 +1,13 @@
-<p>Is Android: {{ platform.ANDROID }}</p>
-<p>Is iOS: {{ platform.IOS }}</p>
-<p>Is Firefox: {{ platform.FIREFOX }}</p>
-<p>Is Blink: {{ platform.BLINK }}</p>
-<p>Is Webkit: {{ platform.WEBKIT }}</p>
-<p>Is Trident: {{ platform.TRIDENT }}</p>
-<p>Is Edge: {{ platform.EDGE }}</p>
+<p>Is Android: {{platform.ANDROID}}</p>
+<p>Is iOS: {{platform.IOS}}</p>
+<p>Is Firefox: {{platform.FIREFOX}}</p>
+<p>Is Blink: {{platform.BLINK}}</p>
+<p>Is Webkit: {{platform.WEBKIT}}</p>
+<p>Is Trident: {{platform.TRIDENT}}</p>
+<p>Is Edge: {{platform.EDGE}}</p>
+<p>Is Safari: {{platform.SAFARI}}</p>
 
 <p>
   Supported input types:
-  <span *ngFor="let type of supportedInputTypes">{{ type }}, </span>
+  <span *ngFor="let type of supportedInputTypes">{{type}}, </span>
```

Comment:
In retrospect, we should've probably just done `supportedInputTypes.join(', ')` but it's not a big deal since it's the dev app.

Reply:
Agreed. I got confused when I saw this.